src/lib/sentry-sanitizer.ts

```diff
@@ -0,0 +1,137 @@
+import { Blueprint } from '@wp-playground/blueprints';
+import { RunCLIArgs } from '@wp-playground/cli';
+
+/**
+ * Sanitizes a Blueprint step to remove sensitive data while keeping useful debugging info.
+ */
+function sanitizeBlueprintStep( step: Blueprint[ 'steps' ][ number ] ): Record< string, unknown > {
+	const baseStep: Record< string, unknown > = { step: step.step };
+	const stepRecord = step as Record< string, unknown >;
+
+	// For steps that might contain secrets, only include safe fields
+	switch ( step.step ) {
+		case 'login':
+			// Omit password, keep username
+			return { ...baseStep, username: stepRecord.username };
```

Comment:
Do we need the username? 🤔 It could be considered sensitive as well. Maybe we could just track the existence of the `login` step.

Reply:
Good suggestion, adjusted on https://github.com/Automattic/studio/pull/1986/commits/812e12073e105d99a447df571e5bedb08503bb00